ui/admin/tests/e2e/tests/session-recording-ent.spec.js

```diff
@@ -140,11 +136,30 @@ test('Verify session recording can be deleted @ent @aws', async ({ page }) => {
       page.getByRole('alert').getByText('Success', { exact: true }),
     ).toBeVisible();
     await page.getByRole('button', { name: 'Dismiss' }).click();
-
-    // Try to delete Session Recording: expect failure
     await waitForSessionRecordingCli(storageBucket.id);
+
+    // Play back session recording
     await page.getByRole('link', { name: 'Orgs', exact: true }).click();
     await page
+      .getByRole('navigation', { name: 'General' })
+      .getByRole('link', { name: 'Session Recordings', exact: true })
+      .click();
+    await page
+      .getByRole('row', { name: targetName })
+      .getByRole('link', { name: 'View' })
+      .click();
+    await page
+      .getByRole('cell', { name: 'Channel 1' })
+      .locator('..')
+      .getByRole('link', { name: 'Play' })
+      .click();
+    await page.locator('div.session-recording-player').hover();
+    await page.locator('.ap-playback-button').click();
```

Comment:
Definitely not the best way to access these elements, but they are the only ones available. I *think* we technically could add like a `test-id` to `session-recording-player`, but wasn't sure if it was worth it. I *think* the playback button is out of our control.

Reply:
Leaving this for now.